using_openshift/scheduler.adoc

```diff
@@ -0,0 +1,77 @@
+= Scheduler
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+The Kubernetes pod scheduler is responsible for scheduling new pods onto minions within the cluster.  It reads data from the pod and tries to find a minion that is a good fit based on configured policies.  It is completely independent and exists as a standalone/pluggable solution.  It does not modify the pod and just creates a binding for the pod that ties the pod to the particular minion. 
+
+== Generic Scheduler
+The existing generic scheduler is the default platform-provided scheduler "engine" that selects a minion to host the pod in a 3-step operation: 
+
+=== Step 1: Filter the minions 
+The available minions are filtered based on constraints/requirements specified.  This is done by running each of the the minion through the list of filter functions, called 'predicates'.
+
+=== Step 2: Prioritize the filtered list of minions
+This is achieved by passing each minion through a series of 'priority' functions that assign it a score between 0 - 10, with 0 indicating a bad fit and indicating a good fit to host the pod.  The scheduler configuration can also take in a simple "weight" (positive numeric value) for each priority function.  The minion score provided by each priority function is multiplied by the "weight" (default weight is 1) and then combined by just adding the scores for each minion provided by all the priority functions.  This weight attribute can be used by administrators to give higher importance to some priority functions. 
```

Comment:
A tiny error "with 0 indicating a bad fit and indicating a good fit to host the pod"